transport/server.cc

```diff
@@ -317,26 +317,30 @@ cql_server::do_accepts(int which, bool keepalive, socket_address server_addr) {
             auto conn = make_shared<connection>(*this, server_addr, std::move(fd), std::move(addr));
             ++_stats.connects;
             ++_stats.connections;
-            // Move the processing into the background.
+
+            // Move the "advertising" and processing into the background.
```

Comment:
Changelog should explain why this is a good thing.

Reply:
I don't know if ithis is a good thing!

Reply:
The motivations are:
- Allow the client application to establish connections ASAP and not "wait" for the completion of the system.clients update (we are seeing that when nodes are loaded connections 
- Do not fail connection setup in case system.clients update timeouts/fails

Reply:
You shouldn't write a patch if you don't know if it's good or not.